dub.json

```diff
@@ -37,9 +37,37 @@
         "source/scpp/build/numeric.o",
         "source/scpp/build/uint128_t.o"
     ],
+    "sourceFiles-windows": [
+        "source/scpp/build/BallotProtocol.obj",
+        "source/scpp/build/ByteSliceHasher.obj",
+        "source/scpp/build/DSCPUtils.obj",
+        "source/scpp/build/DUtils.obj",
+        "source/scpp/build/HashOfHash.obj",
+        "source/scpp/build/Hex.obj",
+        "source/scpp/build/KeyUtils.obj",
+        "source/scpp/build/LocalNode.obj",
+        "source/scpp/build/Logging.obj",
+        "source/scpp/build/Math.obj",
+        "source/scpp/build/NominationProtocol.obj",
+        "source/scpp/build/QuorumSetUtils.obj",
+        "source/scpp/build/SCP.obj",
+        "source/scpp/build/SCPDriver.obj",
+        "source/scpp/build/SHA.obj",
+        "source/scpp/build/SecretKey.obj",
+        "source/scpp/build/Slot.obj",
+        "source/scpp/build/StrKey.obj",
+        "source/scpp/build/crc16.obj",
+        "source/scpp/build/jsoncpp.obj",
+        "source/scpp/build/marshal.obj",
+        "source/scpp/build/numeric.obj",
+        "source/scpp/build/uint128_t.obj",
+        "lib/win64/sodium.lib"
+    ],
 
-    "dflags": [ "-extern-std=c++14" ],
-    "lflags": [ "-lsodium", "-lstdc++", "-lsqlite3" ],
+    "dflags-posix":   [ "-extern-std=c++14"],
+    "dflags-windows": [ "-extern-std=c++14"],
```

Comment:
You can merge them then :)

Reply:
Yes, it's applied

Reply:
Oh..."lflags": [ "-lsodium", "-lstdc++", "-lsqlite3" ],   delete this?

Reply:
I misread the changed line.